JASP-Engine/JASP/R/descriptives.R

```diff
@@ -808,29 +808,30 @@ Descriptives <- function(jaspResults, dataset, options) {
 
 
 .descriptivesFrequencyPlots <- function(dataset, options, variable) {
+  
   if (options$splitby != "" ) {
     # return a collection
     split <- names(dataset)
 
     plotResult <- createJaspContainer(title=variable)
-    plotResult$dependOn(options="splitby", optionContainsValue=list(variables=variable))
+    plotResult$dependOn(options=c("splitby", "binWidthType"), optionContainsValue=list(variables=variable))
```

Comment:
```suggestion
if (is.factor(dataset[[variable]])))
  plotResult$dependOn(options=c("splitby"), optionContainsValue=list(variables=variable))
else
  plotResult$dependOn(options=c("splitby", "binWidthType"), optionContainsValue=list(variables=variable))
```
The thing here is that the histograms of categorical variables don't change when you change the breaks type. So that shouldn't be a dependency.